docs/source/azure/configuration/hostingoptions.rst

```diff
@@ -52,21 +55,25 @@ When deploying an Azure function the following ARM resources are required and cr
 
 Functions can be deployed in either a dedicated Application Service Plan (Basic, Standard or Premium) or in a Consumption plan.
 More details on the different hosting models offered by Azure Functions can be found in the `Azure Functions documentation <https://docs.microsoft.com/en-us/azure/azure-functions/functions-scale>`_.
-By default, we will run all Custodian policies using the Consumption hosting model. (i.e. skuTier=dynamic)
-Linux Consumption is currently only available in the following regions: East Asia, East US, West Europe, and West US
+By default, Custodian policies are run using the Consumption hosting model. (i.e. skuTier=dynamic)
 
-A dedicated plan can service multiple Function Applications.  If you provide the same servicePlanName with all policies or
-use the default name then only new Function Applications will be created during deployment, all using the same
-shared plan resources.
+Linux Consumption plans are not available in all regions. The supported regions are:
```

Comment:
I was not able to find a doc that listed the available regions for the linux consumption plan. I got the list by creating a new application service plan in the portal and seeing which regions were available. I can remove the list and just leave the note that it will not be supported in all regions and that the user should check their region

Reply:
Change made. Informs the user that an error will be thrown if the region is not supported